src/Engine/Objects/CombinedSkillValue.h

```diff
@@ -26,17 +27,17 @@ inline Segment<PLAYER_SKILL_MASTERY> SkillMasteries() {
     return Segment(PLAYER_SKILL_MASTERY_FIRST, PLAYER_SKILL_MASTERY_LAST);
 }
 
-inline PLAYER_SKILL_LEVEL GetSkillLevel(PLAYER_SKILL skill_value) {
-    return skill_value & 0x3F;
+inline PLAYER_SKILL_LEVEL GetSkillLevel(const PLAYER_SKILL skill_value) {
+    return skill_value & 63;
```

Comment:
In my experience for logic operations it's better use hex immediate values.
Because when I look at 0x3F I can quickly figure out that it's "0011 1111". But then number is decimal it becomes much harder.